### ISSUE TITLE
[BUG] doc site is down

### ISSUE THREAD

**Issue by ryan-williams** [wasi-master/pypi-command-line#28]
https://wasimaster.me/pypi-command-line/

![image](https://github.com/user-attachments/assets/e0090759-be42-457a-ba90-29e4bc7b3a66)

There's broken links on this repo's main page, https://wasimaster.me/, https://pypi.org/project/pypi-command-line/, and Google search.

**Comment by wasi-master** [comment]
Thanks, I'll look into it.

**Comment by wasi-master** [comment]
I for the life of me can't seem to figure out what is causing this. I did everything. Now the last thing I'm going to do is stop using GitHub Pages and transfer to readthedocs.

**Comment by ryan-williams** [comment]
Sorry to hear that! I have some experience with GitHub Pages, if you want to "rubber-duck" what you are seeing/debugging…

Looks like your main domain is hosted at Digital Ocean?
```bash
dig +short wasimaster.me
# clownfish-app-8ufys.ondigitalocean.app.
# 172.66.0.96
# 162.159.140.98
```

**Comment by wasi-master** [comment]
> Sorry to hear that! I have some experience with GitHub Pages, if you want to "rubber-duck" what you are seeing/debugging…
> 
> Looks like your main domain is hosted at Digital Ocean?
> ```bash
> dig +short wasimaster.me
> # clownfish-app-8ufys.ondigitalocean.app.
> # 172.66.0.96
> # 162.159.140.98
> ```

Yeah the main wasimaster.me is currently hosted at digitalocean. I used to host it with GitHub Pages in the past but a few days ago switched it to digital ocean. I believe I did something wrong while removing it from GitHub Pages and for that reason all subdomains are redirecting even though the [main page isn't](https://wasi-master.github.io). But another weird thing is One of my other projects' documentation which is hosted with GitHub Pages the same way as this one [works fine](https://wasi-master.github.io/todot/).
